src/apns_connection.erl

```diff
@@ -423,24 +567,43 @@ proxy(#{proxy_info := Proxy}) ->
 proxy(_) ->
   undefined.
 
+-spec default_max_gun_streams(connection()) -> non_neg_integer() | infinity.
+default_max_gun_streams(Setts) ->
+    case type(Setts) of
+        token -> 1; %% at start, for token we should set 1
+        _     -> 100
+    end.
```

Comment:
The indentation in this function doesn't seem to match the rest of the module.

Reply:
fixed